tests/components/Microsoft.DotNet.UpgradeAssistant.MSBuild.Tests/MSBuildTestHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Evaluation;
+
+namespace Microsoft.DotNet.UpgradeAssistant.MSBuild.Tests
+{
+    public static class MSBuildTestHelper
+    {
+        public static string GetTestFolder(string folderName)
+        {
+            var testFolderPath = Path.Combine(Path.GetTempPath(), folderName);
+            if (!Directory.Exists(testFolderPath))
+            {
+                Directory.CreateDirectory(testFolderPath);
+            }
+
+            return testFolderPath;
+        }
+
+        public static string AddFileToTestFolder(string folderName, string fileName, string fileContents)
+        {
+            var testFolderPath = GetTestFolder(folderName);
+            string file = Path.Combine(testFolderPath, fileName);
+            using (var sw = new StreamWriter(file))
+            {
+                sw.WriteLine(fileContents);
+            }
+
+            return file;
+        }
+
+        public static Microsoft.Build.Evaluation.Project LoadProject(string path)
+        {
+            ProjectCollection collection = ProjectCollection.GlobalProjectCollection;
+            return collection.LoadProject(path);
+        }
+
+        public static string CreateProject(string folderName)
```

Comment:
How would this be customized for new tests? This is hardcoding a project file that new tests would want to define differently probably.

Reply:
Changed the method so now new tests can customize how they want the project content to be created.